blitz_api/management/commands/assign_past_tomatoes.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+
+from workplace.models import Reservation
+
+
+class Command(BaseCommand):
+    help = 'Assign past tomatoes. If tomatoes are already assigned, do ' \
+           'nothing. Else add an entry in tomato.Tomato. By default assign ' \
+           'all sources. You can specify the source in the parameters.' \
+           'List of sources: Retreat, Timeslot'
+
+    def add_arguments(self, parser):
+        # Optional arguments
+        parser.add_argument('source', type=str, nargs='?', default=None)
+
+    @staticmethod
+    def assign_tomatoes_past_retreat():
+        """
+        Assign all tomatoes for each past date of retreats to user with
+        active reservation.
+        """
+        from retirement.tasks import assign_retreat_tomatoes
+        assign_retreat_tomatoes()
+
+    @staticmethod
+    def assign_tomatoes_past_timeslot():
+        """
+        Assign all tomatoes for each past timeslots to user with
+        active reservation.
+        """
+        for resa in Reservation.objects.filter(is_present=True):
+            resa.assign_tomatoes()
```

Comment:
Old timeslot should credit tomato even without presence, future timeslot only will check presence